.clang-tidy

```diff
@@ -32,7 +32,9 @@ WarningsAsErrors:   'bugprone-*,
 # -misc-unused-parameters - too many errors, fix later
 
 # Include all headers in the project
-HeaderFilterRegex: 'concord-bft'
+# Except for protobuf/gRPC generated header files
+# Note: '(?<!' starts a negative look behind expression
```

Comment:
Good comment!

Reply:
I took me quite some time to figure this out so I thought it is worth a comment.